### PR TITLE
Upgrade to Protocol 7 - including new fonts (#7102)

### LINKED ISSUE #7102: Update Protocol pages to use Inter
The new font stack for the new brand guidelines is available and mockups are being delivered with the font.

We are blocked on using the new fonts on mozilla.org until https://github.com/mozilla/protocol/issues/342 is resolved.

The rollout plan is:
- [x] Limit Zilla to Mozilla pages
- [x] Remove Zilla from the default font stack
- [x] Make the default font for headings and body copy Inter
- [x] Update the Firefox theme to use ~Sharp Sans~ for headings
- [ ] Import Protocol updates into bedrock

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Update to Protocol 7
  - review use of text size mixins to see where decrease needed
  - update mixin names to use new naming (font-base, font-mozilla)
- Add Inter to font folder
- Remove system font include and usages
- Remove references to Open Sans from all Protocol files
- Update font used on logos in `media/img/logos/firefox/`
  - Update image sizes in places where these logos are used
- Remove "confetti" and "blob" images

## Issue / Bugzilla link
Close #7102

## Testing
You will need to run `make build` to pull the new version of Protocol in.

Impacted pages on demo:
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/ (check homepage in both Firefox and Chrome)
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/accounts/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/campaign/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/concerts/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/election/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/enterprise/sla/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/features/adblocker/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/fights-for-you/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/download/thanks/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/40.0/firstrun/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/pocket/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/this-browser-comes-highly-recommended/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/60.0.2/releasenotes/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/67.0a1/releasenotes/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/releases/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/63.0/whatsnew/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/64.0/whatsnew/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/65.0/whatsnew/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/66.0/whatsnew/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/firefox/67.0/whatsnew/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/foundation/annualreport/2017/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/about/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/technology/browser-history/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/technology/incognito-browser/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/about/policy/lean-data/
- https://bedrock-demo-protocol.oregon-b.moz.works/locales/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/technology/what-is-a-browser/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/privacy/email/
- https://bedrock-demo-protocol.oregon-b.moz.works/en-US/privacy/firefox/

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Review can begin but hold off on merge until design review is finished.

File: media/css/protocol/base/fonts/_firefox.scss [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+
 
 @font-face {
-    font-family: 'Sharp Sans';
+    font-family: 'Firefox Sans';
     font-weight: normal;
     font-style: normal;
     font-display: swap;
     src: url('https://assets.mozilla.net/fonts/FirefoxSharpSans-Medium.woff2') format('woff2'),
          url('https://assets.mozilla.net/fonts/FirefoxSharpSans-Medium.woff') format('woff');
```
Comment (jgmize) [inline media/css/protocol/base/fonts/_firefox.scss]: Please remove all references to these fonts: https://github.com/mozmeao/infra/issues/1118#issuecomment-498386788

File: media/css/protocol/protocol.scss [hunk 2]
```diff
@@ -5,6 +5,9 @@
 $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
+// Enable Firefox Sans on Bedrock
+@import 'base/fonts/firefox';
```
Comment (jgmize) [inline media/css/protocol/protocol.scss]: Please remove all references to Firefox Sans from this PR: https://github.com/mozmeao/infra/issues/1118#issuecomment-498386788

File: media/css/protocol/base/fonts/_firefox.scss (hunk 1 above)
Comment (stephaniehobson) [reply media/css/protocol/base/fonts/_firefox.scss]: Removed.

File: media/css/protocol/protocol.scss (hunk 2 above)
Comment (stephaniehobson) [reply media/css/protocol/protocol.scss]: Removed.

File: media/css/protocol/base/fonts/_firefox.scss (hunk 1 above)
Comment (jgmize) [reply media/css/protocol/base/fonts/_firefox.scss]: Thanks @stephaniehobson

**Comment by stephaniehobson** [comment]
Updated.

**Pull request merged**